AlphaWallet/Tokens/Types/TokensDataStore.swift

```diff
@@ -406,12 +406,11 @@ enum TokenUpdateAction {
                 }
             }
 
-            result = true
-
-            realm.delete(tokenObject.balance)
-            tokenObject.balance.append(objectsIn: newBalance)
+            tokenObject.balance.removeAll()
```

Comment:
Do we still need `result = true` ?

Reply:
for now yes, but i got another non merged pr, where it not needed